index.js

```diff
@@ -28,6 +36,16 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
 
+// let current_thirst_level = 12;
+
+// function drinkWater() {
+//   if (current_thirst_level === 12) {
+//     console.log("Man I sure am thirsty");
+//     current_thirst_level -= 1
+//   }
+//   console.log("Ahh that hits the spot");
+//   current_thirst_level
+// }
 // console.log(drinkWater(12))
```

Comment:
A couple of notes for this one: 

- There's no need for the `current_thirst_level` variable - the idea is just to pass it in when calling the function: `drinkWater(12)`
- Also no need for the `if` statement; based on the way the Ruby example code works, there's no condition where the thirst level won't decrement
- Lastly, this function is meant to **return** the current_thirst_level, so you need to use the `return` keyword.

Here's a working example:

```js
function drinkWater(currentThirstLevel) {
    console.log('man I sure am thirsty')
    currentThirstLevel -=1
    console.log('ahh that hits the spot')
    return currentThirstLevel
}
console.log(drinkWater(12))
```